os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/hci/hci_uart.c

```diff
@@ -80,7 +80,7 @@ T_HCI_UART *hci_uart_obj;
       #define HCI_UART_IRQ  UART1_IRQ
 #endif
 
-#define HCIUART_IRQ_PRIO    10
+#define HCIUART_IRQ_PRIO    6
```

Comment:
It using 10, 6, 3 then definition should be here, below is an example.
```
#define HCIUART_PRIO_HIGH 10
#define HCIUART_PRIO_MID 6
#define HCIUART_PRIO_LOW 3
...
#define HCIUART_IRQ_PRIO HCIUART_PRIO_HIGH
...
#define TRACE_TASK_PRIO HCIUART_PRIO_LOW
```

Reply:
It will be fix after the PR, I think we can just use 6.